src/test/unit/profile-conversion.test.js

```diff
@@ -98,4 +102,33 @@ describe('converting Google Chrome profile', function() {
     }
     expect(profile).toMatchSnapshot();
   });
+
+  it('successfully imports a single CpuProfile, e.g. from node', async function() {
+    const fs = require('fs');
+    const buffer = fs.readFileSync(
+      'src/test/fixtures/upgrades/test.chrome-unchunked.json'
+    );
+    const events: TracingEventUnion[] = JSON.parse(buffer.toString('utf8'));
+
+    // Use a for loop to look this up, as Flow is happier than a .find().
+    let cpuProfile: CpuProfileEvent | void;
+    for (const event of events) {
+      if (event.name === 'CpuProfile') {
+        cpuProfile = event;
+        break;
+      }
+    }
+
+    if (!cpuProfile) {
+      throw new Error('Could not find a CPU profile');
+    }
```

Comment:
nit: it would be nice to have a comment explaining what we're doing (essentially extracting one cpu profile from the array in the file, right?)

Reply:
Yup, I added a comment and a lexical scope to make it clearer.